cli/core/docs/cli/cli-architecture.md

```diff
@@ -135,9 +171,19 @@ tanzu context use mgmt-cluster
 
 ## Target
 
-The Tanzu CLI supports two targets (context types): `kubernetes`, `mission-control`. This is currently backwards compatible, i.e., the plugins are still available at the root level. In addition to that, we also have contextual plugins grouped under the target.
+Target is a top level entity used to make the control plane, that a user is interacting against, more explicit in command invocations.
+This is done by creating a separate target specific command under root level command for Tanzu CLI. e.g. `tanzu <target>`
+
+The Tanzu CLI supports two targets: `kubernetes`, `mission-control`. This is currently backwards compatible, i.e., the plugins are still available at the root level.
```

Comment:
The second sentence is hard to understand.  How about replacing the second sentence with:
"To respect backwards-compatibility, when no target command is specified, the `kubernetes` target is used automatically.